### PR TITLE
Metric-webApp-CryptoPanel pull request

### PR DESCRIPTION
**Pull request by AleWaweru**
- Fetched coin and currency API from coingecko.com
- Installed required dependencies such as Redux-React, Redux toolkit, and linters.
- Did unit test using render and snapshot

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,171 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+
```
Comment (DammyShittu) [inline README.md]: - [ ] Kindly remove this section description from your documentation.

**Review by DammyShittu — COMMENTED** [review]
Hi @AleWaweru,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### Kindly remove the section description in the Authors section.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md (hunk 1 above)
Comment (AleWaweru) [reply README.md]: done

File: README.md [hunk 2]
```diff
@@ -1,70 +1,169 @@
...
+<p>I would like to thank <a href = "https://www.behance.net/sakwadesignstudio">Nelson Sakwa .</a> the author of the original design.</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```
Comment (mukhammadrizooff) [inline README.md]: - [ ] [REQUIRED] Good job so far! Kindly you have not this  kind of file therefore your `MIT` license displayed nothing, please create build it 🙏🏼 

image

File: src/Components/Main/SearchBar.jsx
Comment (mukhammadrizooff) [inline src/Components/Main/SearchBar.jsx:1]: - [ ] [REQUIRED] Good job so far! Your `Search` functionality is not working on my side and shows errors, please fix them 🙏🏼 

ezgif com-optimize

File: src/Components/Main/CurrentList.css
Comment (mukhammadrizooff) [inline src/Components/Main/CurrentList.css:1]: - [ ] [REQUIRED] Good job so far! I appreciate what you have done but you should give your project `colors` like the design provided by [Behance](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) and you should also create other `elements`. Please implement my suggestion 🙏🏼 

image


image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @AleWaweru, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Correct GitFlow ✔️ 
- Good implementations ✔️
- Descriptive `README` file ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (AleWaweru) [reply README.md]: Done

File: src/Components/Main/SearchBar.jsx
Comment (AleWaweru) [reply src/Components/Main/SearchBar.jsx:1]: I think it was not a  requirement to implement the search button, the instructions were to implement the click/tap on the list Item to display more information.

# When the user clicks (or taps) on a category item, the application navigates to the details page.

File: src/Components/Main/CurrentList.css
Comment (AleWaweru) [reply src/Components/Main/CurrentList.css:1]: done.

**Review by binhussen — APPROVED** [review]
Hi @AleWaweru , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- Sticked Given Design ✔️
- Well Structured Code ✔️
- Used Git flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Components/Main/SearchBar.jsx
Comment (mukhammadrizooff) [reply src/Components/Main/SearchBar.jsx:1]: ### Hi @AleWaweru! I appraciate your question but in my opinion this was also important thing to implement it and increase your `problem solving` skills 😃 

image

**Pull request merged**